### ISSUE TITLE
Trigger rebuilds on repositories

### ISSUE THREAD

**Issue by sarahhodne** [travis-ci/travis-watcher-macosx#8]
How would we go about doing this? I'm assuming it is possible through the API?

**Comment by joshk** [comment]
Yep, it is, but first we need to tackle login and automatic repo filtering based on login.

**Comment by sarahhodne** [comment]
Of course. I was just moving TODOs out of the README.
